content/_data/changelogs/lts.yml

```diff
@@ -3604,6 +3604,213 @@
+      Indicate when security issues would be addressed by an update in plugin manager.
+      Sort plugins by popularity on the <em>Available</em> plugin manager tab if the update site provides popularity data.
+      Show plugin categories as labels in the plugin manager instead of grouping them into different table sections.
+      Allow use of multiple space-separated filter terms in plugin manager.
+      Allow linking to plugin manager URLs with pre-filled filter field.
+      Link labels in the plugin manager to pre-filtered lists.
+      Show plugin release date in plugin manager.
+      List plugins that failed to load on the Installed tab of the plugin manager.
+      Highlight in the plugin manager when plugins are looking for new maintainers ("Adopt this plugin").
+
+  # Major user experience improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4546
+    authors:
+    - daniel-beck
+    message: |-
+      Organize entries on the <em>Manage Jenkins</em> page into categories and show them in a grid.
+
+  # Other improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4503
+    authors:
+    - daniel-beck
+    message: |-
+      Remove 'auto refresh' feature, including now obsolete auto refresh telemetry capability.
+
+  # System/Read improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4479
+    issue: 61202
+    authors:
+    - timja
+    references:
+    - issue: 61202
+    - pull: 4479
+    - url: https://www.jenkins.io/blog/2020/05/25/read-only-jenkins-announcement/
+      title: Read-only Jenkins Configuration blog post
+    message: |-
+      Users with extended read permission now get a more read-only looking UI.
+  - type: major rfe
+    category: major rfe
+    pull: 4520
+    issue: 61201
+    references:
+    - issue: 61205
+    - pull: 4524
+    - issue: 61201
+    - pull: 4520
+    - issue: 61203
+    - pull: 4518
+    - issue: 61207
+    - pull: 4530
+    - issue: 61455
+    - pull: 4569
+    - issue: 61208
+    - pull: 4533
+    - issue: 61208
+    - pull: 4685
+    authors:
+    - timja
+    - amuniz
+    - daniel-beck
+    message: |-
+      Allow users with system read permission to view the <em>About Jenkins</em> page, the <em>Manage Plugins</em> configuration, the global security configuration page, the admin monitors, and the system logs.
+  - type: bug
+    category: bug
+    pull: 4622
+    issue: 61812
+    authors:
+    - timja
+    message: |-
+      Distinguish between defined (<em>*****</em>) and undefined (<em>N/A</em>) password on read-only configuration forms for users with <code>Overall/SystemRead</code> or <code>Item/ExtendedRead</code> permissions.
+
+  # Overall/Manage improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4571
+    issue: 61456
+    references:
+    - issue: 61456
+    - pull: 4571
+    - issue: 61453
+    - pull: 4572
+    - issue: 61457
+    - pull: 4570
+    - issue: 61455
+    - pull: 4569
```

Comment:
As above, would remove `pull` references when there are `issue`.

Reply:
Fixed in f46fa4e6a5915dbc6ad65182019e62ea7e4f908d